adapter-protobuf-java/src/main/proto/adapter.proto

```diff
@@ -134,19 +154,30 @@ message ActionResponse {
 // Different actions can require certain input
 enum AttachmentType {
 
+  // Request attachment.
   // Useful for authorization.
   // Try to avoid depending on this if possible as it comes
   // with a lot of GDPR related requirements.
   NORWEGIAN_FODSELSNUMMER = 0;
 
+  // Response attachment. Useful for sending something like a
+  // member number to users of Oslonøkkelen.
   CODE = 1;
 
   reserved 2;
 
+  // Response attachment.
+  // Allows the adapter to send a human readable message back
+  // to the user of the app responsible for issuing the request.
   END_USER_MESSAGE = 3;
 
+  // Response attachment.
+  // Technical error information useful for debugging and troubleshooting.
   ERROR_DESCRIPTION = 4;
 
+  // Response attachment.
+  // Technical information explaining why an action request was denied.
+  // Should only be used for action responses with status = denied.
   DENIED_REASON = 5;
 
 }
```

Comment:
Could be a good idea to split these into two different enums?

Reply:
Agree, I think the idea behind keeping both types in the same enum was that we might have some attachment relevant for both request and response. Anyway, too late to change this now that multiple adapters has implemented this.